okhttp/src/main/java/okhttp3/HttpUrl.java

```diff
@@ -772,6 +772,23 @@ public Builder addPathSegment(String pathSegment) {
       return this;
     }
 
+    public Builder addPathSegments(String groupedPathSegments) {
```

Comment:
Yup, that was dumb of me haha. fixing.
I am a bit nervous about the similarity in the method names. There would be no cost using the "plural" method, just the possibility of hiding bad input.